### ISSUE TITLE
Tests fail with Mojolicious 7

### ISSUE THREAD

**Issue by eserte** [jberger/Mojolicious-Plugin-SimpleSlides#4]
Another fail with Mojolicious 7.
A sample report: http://www.cpantesters.org/cpan/report/52a26930-4e40-11e6-9c90-eb5010d7a009

Statistical analysis:

```
****************************************************************
Regression 'mod:Mojolicious'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    15354392830366858.00
[1='eq_6.55']        -0.0000          0.0000      -1.21
[2='eq_6.56']        -0.0000          0.0000      -1.61
[3='eq_6.57']         0.0000          0.0000       0.00
[4='eq_6.58']        -0.0000          0.0000      -2.78
[5='eq_6.59']        -0.0000          0.0000      -1.21
[6='eq_6.60']        -0.0000          0.0000      -1.39
[7='eq_6.61']        -0.0000          0.0000      -6.10
[8='eq_6.62']        -0.0000          0.0000      -1.59
[9='eq_6.66']         0.0000          0.0000       0.00
[10='eq_7.0']        -1.0000          0.0000    -15031106248587852.00

R^2= 1.000, N= 78, K= 11
****************************************************************
```

**Comment by eserte** [comment]
Probably relevant change: https://metacpan.org/source/SRI/Mojolicious-7.0/Changes#L8

**Comment by jberger** [comment]
As with App::MojoSlides, this module really should just be put out of its misery. Thanks for the report anyway though (and the others too)!

**Comment by jberger** [comment]
Fixed by 4c33155c68af55dfaa7fd25f6613da48c0571d33 . Thanks
